deploy/gateway/templates/service.yaml

```diff
@@ -23,5 +23,9 @@ spec:
       port: 443
       targetPort: https
       protocol: TCP
+    - name: metrics
+      port: 9090
+      targetPort: metrics
+      protocol: TCP
```

Comment:
We shouldn't expose the metrics port in the service because that might expose Prometheus metrics externally. Prometheus could be configured to scrape metrics directly from the Gateway pods.